### ISSUE TITLE
Nginx-gen reboot exit 2

### ISSUE THREAD

**Issue by paulunlmtd** [evertramos/docker-compose-letsencrypt-nginx-proxy-companion#50]
Any idea why nginx-gen fails on a reboot, with an exit code of 2? I see the problem referenced in the parent project here: (https://github.com/jwilder/docker-gen/issues/201) with no solution, as of yet.

**Comment by Aschenbecher** [comment]
I have the same problem :( After restarting the docker Service, nginx-gen is not restarting, even with restart: always. No infos in logfiles

**Comment by Aschenbecher** [comment]
Im using compose file from git

**Comment by evertramos** [comment]
Go to the folder where you have the docker-compose file and do `docker-compose up -d`.... sorry for this issue, but is not really related to our compose file.. you might need to go to the @jwilder repo to opne an issue there.

**Comment by Aschenbecher** [comment]
> Go to the folder where you have the docker-compose file and do `docker-compose up -d`.... sorry for this issue, but is not really related to our compose file.. you might need to go to the @jwilder repo to opne an issue there.

yes ok, I added 'docker start nginx-gen' to my backup scipt, will fix it for now, will open an issue at jwilder git, thanks for fast reply!
